source/layouts/layout.erb

```diff
@@ -1,6 +1,16 @@
 <!doctype html>
 <html lang="en" class="no-js">
   <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-127779891-1"></script>
+    <script>
```

Comment:
Might be good to separate this out into its own file and import it here - `layout.erb` is already pretty huge, worried it could get lost.

Suspect there's guides online for doing exactly that in rails, if it's not clear where to start I can help out.

Could also argue it's not that big a deal for the sake of getting this live and feeding us useful data.

Reply:
Sounds clean

Reply:
I like clean

Reply:
I hadn't twigged this was on the product page rather than admin, so this isn't a rails thing, it's a middleman thing.

Should still be straightforward / have info out there though.